Rules/mtbmap.mrules

```diff
@@ -1701,24 +1707,26 @@ rules
 			font-style : normal
 			font-weight : bold
 			text-color : black
+			text-avoid-overlaps : true
 			text-orientation: fixed
 			min-zoom : 15
 			font-size : 14:10; 15:11; 16:12
 			line-width : 0.5
 			painting-cycle : 400
-			text-func : NodeName(e)
+			
```

Comment:
It is better IMO to keep the `text-func : NodeName(e)` as it is common to all sub-categories that use `draw : text`

It will allow removing several `text-func : NodeName(e)` below